packages/sanity/src/core/validation/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './types'
```

Comment:
Are we sure we want to export literally everything? Just double checking since we've been pretty reduced with the number of things we export beneath. 🤔

Reply:
It's only three members, all of them public. But yes, this might be a good time to start limiting the exposed types :)
I'll switch to explicit exports after merging!